project/IntegrationTests.scala

```diff
@@ -0,0 +1,31 @@
+package akka.projections
+
+import de.heikoseeberger.sbtheader.HeaderPlugin.autoImport.headerSettings
+import sbt.Keys.*
+import sbt.*
+
+object IntegrationTests {
+
+  val IntegrationTestTag = Tags.Tag("IntegrationTest")
+
+  def settings: Seq[Def.Setting[_]] =
+    Seq(
+      publish / skip := true,
+      doc / sources := Seq.empty,
+      Test / fork := true,
+      Test / javaOptions ++= {
+        import scala.collection.JavaConverters._
+        // include all passed -Dakka. properties to the javaOptions for forked tests
+        // useful to switch DB dialects for example
+        val akkaProperties = System.getProperties.stringPropertyNames.asScala.toList.collect {
+          case key: String if key.startsWith("akka.") || key.startsWith("config") =>
+            "-D" + key + "=" + System.getProperty(key)
+        }
+        akkaProperties
+      },
+      // run one integration test at a time
```

Comment:
can't we just run all tests sequentially? Do we win much from trying to run some tests in parallel. Log capturing will be messed up when running in parallel, or maybe that works when forking, but do we work ordinary tests?

Reply:
That would probably be fine, there isn't anyway very many unit tests.